docs/en/ingest-management/beats-agent-comparison.asciidoc

```diff
@@ -46,14 +47,16 @@ The following steps will help you determine if {agent} can support your use case
 * `System`
 * `Windows`
 * `Zeek`
-
++
 NOTE: For more details, refer to {integrations-docs}[{integrations}].
++
+. If the integration is available, check <<supported-outputs-beats-and-agent>> to see whether the required output is also supported.
 
-**Step 2:** If the Integration is available, check to see whether the required output is also supported. The supported outputs are outlined in this document ([Supported Outputs](https://docs.google.com/document/d/16qAC3iuNudiiMS0vL5PNHGvD6NQOZ0FVhaUZsKSHavw/edit#heading=h.ntxz032s5u9r))
+. Review <<additional-capabilties-beats-and-agent>> to determine if any features required by your deployment are supported. {agent} should support most of the features available on {beats} and is updated for each release.
 
-**Step 3:** Review https://www.elastic.co/guide/en/fleet/current/beats-agent-comparison.html#additional-capabilties-beats-and-agent[Supported Capabilities] to determine if any features required by your deployment are supported. {agent} should support most of the features available on {beats} and is updated for each release.
+// QUESTION: would we ever correct a typo in anchor? i.e. https://www.elastic.co/guide/en/fleet/current/beats-agent-comparison.html#additional-capabilties-beats-and-agent
 
-If you are satisfied with all three steps, then Elastic Agent is suitable for deployment. If any steps fail your assessment, you should continue to use Beats and review future updates or contact us in the [discuss forum](https://discuss.elastic.co/).
+If you are satisfied with all three steps, then {agent} is suitable for your deployment. However, if any steps fail your assessment, you should continue using {beats}, and review future updates or contact us in the https://discuss.elastic.co/[discuss forum].
```

Comment:
```suggestion
If you are satisfied with all three steps, then {agent} is suitable for your deployment. However, if any steps fail your assessment, you should continue using {beats}, and review future updates or contact us in the {forum}[discuss forum].
```

Reply:
The `forum` attribute is defined in https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc#L109.

Reply:
Thanks for both (think I forgot to fold in one of Eamonn's {agent} corrections in too).

I've been using Atom with Default: 80 too but might need a steer to what to fix or we can discuss/document as a group later.